block-producer/src/server/mod.rs

```diff
@@ -52,19 +52,9 @@ pub async fn serve(config: BlockProducerConfig) -> Result<()> {
 
     let block_producer = api_server::ApiServer::new(api::BlockProducerApi::new(queue.clone()));
 
-    tokio::spawn(async move {
-        queue
-            .run()
-            .instrument(info_span!(target: COMPONENT, "transaction_queue_start"))
-            .await
-    });
+    tokio::spawn(async move { queue.run().await });
 
-    tokio::spawn(async move {
-        batch_builder
-            .run()
-            .instrument(info_span!(target: COMPONENT, "batch_builder_start"))
-            .await
-    });
+    tokio::spawn(async move { batch_builder.run().await });
```

Comment:
Should we put just `info` events here - e.g., `transaction queue started` and `batch builder started`?